setup.py

```diff
@@ -22,16 +23,19 @@
 # $ pip install wheel
 # $ python setup.py bdist_wheel --universal
 # $ twine upload dist/*
-
+requirements = ['mxnet-mkl>=1.1', 'Flask', 'Pillow', 'requests', 'flask-cors',
+                'psutil', 'jsonschema', 'onnx-mxnet>=0.4.2', 'boto3', 'importlib2',
+                'fasteners']
+if platform.system() == 'Linux':
+    requirements[0] = 'mxnet-cu90mkl>=1.1'
```

Comment:
This will work, but will be very easy to break in the future. E.g. someone adds a new dependency, and puts it in the first slot in the requirements vector...
Maybe instead we can do a if/else statement that inserts the right package (cpu/gpu) in the 0-th index?

Reply:
agreed , will improve this